build-scripts/bundle.js

```diff
@@ -70,12 +70,7 @@ module.exports.htmlMinifierOptions = {
   removeComments: true,
   removeRedundantAttributes: true,
   minifyCSS: {
-    level: {
-      1: {
-        all: false,
-        removeWhitespace: true,
-      },
-    },
+    compatibility: "*,-properties.zeroUnits",
```

Comment:
How the hell is that not the same as `replaceZeroUnits: false`??? 🤯

Reply:
Yeah... I'm confused too, but glad I got it working 😅

Reply:
Looks like the zero units issue was actually fixed in `clean-css`, but the Babel plugin isn't keeping things updated.  I'll see if I can do something about that.